packages/core/admin/admin/src/pages/ProfilePage/index.js

```diff
@@ -91,6 +94,28 @@ const ProfilePage = () => {
       });
     },
   });
+  useEffect(() => {
+    const getSSOInfo = async () => {
+      try {
+        const {
+          data: {
+            data
+          }
+        } =  await get('/admin/providers/options');
+
+        if (data) {
+          setSsoInfo(data);
+        }
+      } catch (error) {
+        console.error(error.response);
+        toggleNotification({
+          type: 'warning',
+          message: { id: 'notification.error' },
+        });
+      }
+    }
+    getSSOInfo();
+  }, [get, toggleNotification]);
```

Comment:
Can you please use react-query instead?

Reply:
+1 not sure there's a reason to use the raw `get`?

Reply:
I refactored the code to use react query